management/README.md

```diff
@@ -36,36 +40,49 @@ yubikitVersion=1.0.0-beta05
   <version>1.0.0-beta05</version>
 </dependency>
 ```
-### Using Library <a name="using_lib"></a>
+### Using the Module Library <a name="using_lib"></a>
+
+The **MGMT** module requires the YubiKit core library to detect a `YubikeySession` (see [YubiKit Module](../yubikit/README.md)). Use the session to create:
 
-The **MGMT** module requires the yubikit core library to detect a `YubikeySession` (see [Using YubiKit](../yubikit/README.md)). Use the session to create a `ManagementApplication` to select the MGMT applet. Additionally, use the session to create a  `YubiKeyConfigurationApplication` to customize/personalize the OTP slots or challenge-response.
+a) a `ManagementApplication` to select the MGMT applet
+
+b) a  `YubiKeyConfigurationApplication` to customize/personalize the OTP slots or challenge-response.
 
 ```java
 
     ManagementApplication application = new ManagementApplication(session);
     // run provided command/operation (readConfiguration/writeConfiguration)
-    
-    
+
+
     // HMAC-SHA1 challenge-response
     YubiKeyConfigurationApplication application = new YubiKeyConfigurationApplication(session);
     byte[] response = application.calculateHmacSha1(challenge, Slot.TWO);
-    
+
 ```
 
 ### Using the Demo Application <a name="using_demo"></a>
-This module provides multiple demos.
+This module provides several demos.
+
 
 #### Management functionality
-1. Select "YubiKey Settings" pivot in navigation drawer
-1. Turn off/on YubiKey applets and transports (YubiKey Series 5+ required)
-   Note: Use this demo to emulate various YubiKey configurations, e.g. disable the NFC transport to emulate a USB only YubiKey. Verify your application behaves as expected with the various YubiKey configurations.
+YubiKey Series 5+ required. Use this demo to emulate various YubiKey configurations such as disabling the NFC transport to emulate a USB-only YubiKey. Verify your application behaves as expected with the various YubiKey configurations.
+
+1. Select the *YubiKey Settings* pivot in the navigation drawer.
+2. Turn off/on YubiKey applets and transports
+
 
 #### Programming OTP slots
-Things to consider before running this demo: 
-* By default the slot ONE is programmed with YubiOTP secret. Overriding first slot means loosing this secret.
-* YubiOTP codes that were generated on key can be read using "Yubico OTP demo" pivot. Refer to the [OTP module](../otp/README.md) to learn more.
+Before running this demo, be aware that:
+* By default slot ONE is programmed with the YubiOTP secret. Overwriting this first slot means losing this secret.
+* YubiOTP codes that were generated on the YubiKey can be read using the *Yubico OTP demo* pivot. Refer to the [OTP Module](../otp/README.md) to learn more.
```

Comment:
I'm not familiar with this usage of the word "pivot", and I can't really find any similar usage by Googling, so I'm not sure where it comes from. It seems to refer to the different "parts" of the demo app, maybe we can just use _demo_ for this? "... using the *Yubico OTP* demo", and so on?

Reply:
I know "pivot" only as something the company does when we change direction, and in the context of Excel - so I'm happy to get rid of it.